src/node_sqlite.cc

```diff
@@ -461,13 +461,15 @@ DatabaseSync::DatabaseSync(Environment* env,
                            Local<Object> object,
                            DatabaseOpenConfiguration&& open_config,
                            bool open,
-                           bool allow_load_extension)
+                           bool allow_load_extension,
+                           int timeout)
```

Comment:
Can you make `timeout` part of `DatabaseOpenConfiguration`. I think that means it doesn't need to be part of this class anymore.

Reply:
sure